src/features/twitter/services/login-twitter-service.test.ts

```diff
@@ -0,0 +1,34 @@
+import jwt from 'jsonwebtoken';
+import type { Mock } from 'vitest';
+
+import { LoginTwitterService } from './login-twitter-service';
+
+vi.mock('jsonwebtoken', () => ({
+  default: {
+    verify: vi.fn(() => ({ userId: '123' })),
+  },
+}));
+
+describe('LoginTwitterService', () => {
+  let sut: LoginTwitterService;
+  let mockVerify: Mock;
+  const jsonwebtoken = jwt;
+
+  beforeEach(() => {
+    sut = new LoginTwitterService();
+    mockVerify = vi.fn(() => ({ userId: `123` }));
+    jsonwebtoken.verify = mockVerify;
+  });
+
+  it('should return the generated auth URL', () => {
```

Comment:
eu removeria os should e deixaria apenas 
```suggestion
  it('returns the generated auth URL', () => {
```